src/Domain/VehicleFleet.php

```diff
@@ -39,32 +39,34 @@ public function parkVehicle(string $platenumber, Location $where, \DateTimeInter
         $vehicle->park($where, $when);
     }
 
-    public function isVehiclePartOf(string $platenumber)
+    public function isVehiclePartOf(string $platenumber): bool
```

Comment:
this no longer will work since you throw an exception in the `vehicleWithPlatenumber`. We should have a look why test still pass.